app/src/main/assets/ime/text/symbols/mod/cjk.json

```
@@ -0,0 +1,55 @@
+{
+  "type": "symbols/mod",
+  "name": "cjk",
+  "label": "CJK",
+  "authors": [ "waelwindows" ],
+  "direction": "ltr",
+  "arrangement": [
+    [
+      { "code": -203, "label": "view_symbols2", "type": "system_gui" },
+      { "code":    0, "type": "placeholder" },
+      { "code":   -5, "label": "delete", "type": "enter_editing" }
+    ],
+    [
+      { "code": -201, "label": "view_characters", "type": "system_gui" },
+      { "$": "char_width_selector",
+        "full": { "code": 12289, "label": "、", "popup": {
+            "main": { "code":   44, "label": "," }
+          }
+        },
+        "half": { "code": 65380, "label": "､", "popup": {
+            "main": { "code":   44, "label": "," }
+          }
+        }
+      },
+      { "code": -205, "label": "view_numeric_advanced", "type": "system_gui" },
+      { "code": 12288, "label": "空白" },
+      { "code": -9701, "label": "char_width_switcher", "popup": {
```

It's the key type once again, character keys are not allowed to render symbols, while other types do:

```suggestion
      { "code": -9701, "label": "char_width_switcher", "type": "system_gui", "popup": {
```

This is just one key I suggested the change, although you have to do this for all switcher keys, also when they are in the popup